docs_app/content/guide/operators.md

```diff
@@ -296,41 +288,60 @@ function discardOddDoubleEven() {
 
 It is more complicated, but if you have to write an operator that cannot be made from a combination of existing operators (a rare occurrance), you can write an operator from scratch using the Observable constructor, like this:
 
-
 ```ts
-import { Observable } from 'rxjs';
+import { Observable, of } from 'rxjs';
 
 function delay(delayInMillis) {
-  return (observable) => new Observable(observer => {
-    // this function will be called each time this
-    // Observable is subscribed to.
-    const allTimerIDs = new Set();
-    const subscription = observable.subscribe({
-      next(value) {
-        const timerID = setTimeout(() => {
-          observer.next(value);
-          allTimerIDs.delete(timerID);
-        }, delayInMillis);
-        allTimerIDs.add(timerID);
-      },
-      error(err) {
-        observer.error(err);
-      },
-      complete() {
-        observer.complete();
-      }
-    });
-    // the return value is the teardown function,
-    // which will be invoked when the new
-    // Observable is unsubscribed from.
-    return () => {
-      subscription.unsubscribe();
-      allTimerIDs.forEach(timerID => {
-        clearTimeout(timerID);
+  return (observable) =>
+    new Observable((subscriber) => {
+      // this function will called each time this
```

Comment:
Somehow `be` is missing again, probably a bad copy/paste. There was a PR that added it here, let's not miss it again 😃